rviz_assimp_vendor/CMakeLists.txt

```diff
@@ -4,15 +4,9 @@ project(rviz_assimp_vendor)
 find_package(ament_cmake REQUIRED)
 find_package(ament_cmake_vendor_package REQUIRED)
 
-# Override ON so that the following CMake logic in assimp 5.0.1 and older
-# doesn't result in a CMake warning: if(ON)
-#   https://github.com/ros2/rviz/issues/524
-#   https://bugs.launchpad.net/ubuntu/+source/assimp/+bug/1869405
-set(ON 1)
-
 # TODO: Switch to version range in find_package in CMake 3.19
 find_package(assimp QUIET)
-if(NOT assimp_FOUND OR "${assimp_VERSION}" VERSION_LESS 5.2.0)
+if(NOT assimp_FOUND OR "${assimp_VERSION}" VERSION_LESS 5.2.2)
```

Comment:
I can't remember why we chose `5.2.0` instead of `5.2.2` here.  @cottsay do you remember at all?

Reply:
I don't think they changed to patch release in the CMake logic when they shipped.

Reply:
There's an argument that we should only require the same major/minor anyway, but we can discuss that OOB.

Reply:
Confirmed:
```
root@157c919d4ef8:/# dpkg-query --show libassimp-dev
libassimp-dev:amd64	5.2.2~ds0-1
root@157c919d4ef8:/# grep 5\.2 /usr/lib/x86_64-linux-gnu/cmake/assimp-5.2/assimpConfigVersion.cmake 
set(PACKAGE_VERSION "5.2.0")
  if("5.2.0" MATCHES "^([0-9]+)\\.")
    set(CVF_VERSION_MAJOR "5.2.0")
root@157c919d4ef8:/# grep -i codename /etc/lsb-release 
DISTRIB_CODENAME=jammy
```

Reply:
Also, you can see in the `Rpr` job on this PR that it's building from source on Jammy, which isn't what we want to do.

Requiring `5.2.0` is the best we can do without jumping through hoops.

Reply:
OK, yeah, agreed.  In that case, I'll close out this PR.  Thanks for taking a look, it is much appreciated!